frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings/SdkInfoAlert.tsx

```diff
@@ -0,0 +1,65 @@
+import type { ReactNode } from "react";
+import { match } from "ts-pattern";
+import { jt, t } from "ttag";
+
+import { useSetting } from "metabase/common/hooks";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { PLUGIN_EMBEDDING_SDK } from "metabase/plugins";
+import { Alert, Icon, Text } from "metabase/ui";
+
+import { useEmbeddingSettingsLinks } from "./sdk";
+
+const getInfoText = (value: ReactNode) => {
+  return jt`You can test Embedded analytics SDK on localhost quickly by using API keys. To use the SDK on other sites, ${value}`;
+};
+
+export const SdkInfoAlert = () => {
+  const { implementJwtUrl, switchMetabaseBinariesUrl, upgradeUrl } =
+    useEmbeddingSettingsLinks();
+  const isHosted = useSetting("is-hosted?");
+  const isEE = PLUGIN_EMBEDDING_SDK.isEnabled();
+
+  const upgradeLink = (
+    <ExternalLink href={upgradeUrl}>{t`upgrade to Metabase Pro`}</ExternalLink>
+  );
+  const jwtLink = (
+    <ExternalLink href={implementJwtUrl}>{t`implement JWT SSO`}</ExternalLink>
+  );
+
+  const switchLink = (
+    <ExternalLink href={switchMetabaseBinariesUrl}>
+      {t`switch Metabase binaries`}
+    </ExternalLink>
+  );
+
+  const alertText = match({ isEE, isHosted })
```

Comment:
I think I gave the same suggestion on some other PR, but I'd inline the variables as it seems they're used only once.
with `match({isEE:PLUGIN_EMBEDDING_SDK.isEnabled(),...})` I don't have to go to the top of the file and see what we mean by `isEE`